modules/nf-core/skani/dist/main.nf

```diff
@@ -0,0 +1,50 @@
+process SKANI_DIST {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/skani:0.2.1--h4ac6f70_0':
+        'biocontainers/skani:0.2.1--h4ac6f70_0' }"
+
+    input:
+    tuple val(meta) , path(query)
+    tuple val(meta2), path(reference)
+
+    output:
+    tuple val(meta), path("${prefix}.tsv")  , emit: dist
+    path "versions.yml"                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}_${meta2.id}"
+    """
+    skani \\
+        dist \\
+            -q ${query} \\
+            -r ${reference} \\
+            -o ${prefix}.tsv \\
+            -t ${task.cpus} \\
+            ${args}
```

Comment:
I saw it has `-v` and `-t` do either include useful stats [^1] for potentai l MultiqC? stuff

[^1]: https://nf-co.re/docs/contributing/nfcore_component-specifications/modules#110-capturing-stdout-and-stderr

Reply:
Just checked and tbh the output is kind of meaningless to me...
`[00:00:00.000] (7f54a622e800) INFO   skani dist -q genome.fasta.gz -r contigs.fasta -o test_test2.tsv -t 2 --debug
[00:00:00.000] (7f54a622e800) DEBUG  Using C125 regression model.
[00:00:00.001] (7f54a622e800) INFO   Learned ANI mode detected. ANI may be adjusted according to a regression model trained on MAGs.
[00:00:00.001] (7f54a622e800) INFO   Generating sketch time: 0.00141721
[00:00:00.001] (7f54a622e800) DEBUG  Ref seeds len 230, Query seeds len 271, Anchors 228, Seeds hit query 224, Est 0.9873820971742308, Ref_file contigs.fasta, Query_file genome.fasta.gz
[00:00:00.001] (7f54a622e800) DEBUG  Query genome.fasta.gz Ref contigs.fasta - ANI 0.9993965571153403, +/- = 0/1. 
[00:00:00.001] (7f54a622e800) INFO   ANI calculation time: 0.00033303`